.scripts/copyright-notice

```diff
@@ -0,0 +1,3 @@
+// Copyright 20XX - See NOTICE file for copyright holders.
+//
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
The SPDX identifier should be the first line of any file. It is also the only line needed.
```suggestion
// SPDX-License-Identifier: Apache-2.0
```

Please also update all source file headers...

Reply:
Done